lib/post/utils/post.dart

```diff
@@ -39,6 +39,29 @@ Future<bool> markPostAsRead(int postId, bool read) async {
   return markPostAsReadResponse.isSuccess();
 }
 
+/// Logic to mark multiple posts as read
+Future<List<int>> markPostsAsRead(List<int> postIds, bool read) async {
+  Account? account = await fetchActiveProfileAccount();
+  LemmyApiV3 lemmy = LemmyClient.instance.lemmyApiV3;
+
+  if (account?.jwt == null) throw Exception('User not logged in');
+
+  List<int> failed = [];
+  // TODO: Lemmy 0.19 support postIds in MarkPostAsRead
+  //       Check what version the server is, and use that if we can
```

Comment:
Check out the `supportsFeature` method in `lemmy_client.dart` if you want to make a version-specific call.